zz/zznome.sh

```diff
@@ -38,12 +38,20 @@ zznome ()
 		case "$2" in
 			origem     ) sed -n '/Qual a origem do nome /{s/Qual a o/O/;p;}' ;;
 			significado) sed -n '/Qual o significado do nome /{s/Qual o s/S/p;}' ;;
-			letra      ) sed -n '/Analise da Primeira Letra do Nome:/,/Sua marca no mundo!/{/Sua marca no mundo!/d;p;}' ;;
-			marca      ) sed -n '/Sua marca no mundo!/,/Significado - Numerologia - Expressão/{/Significado - Numerologia - Expressão/d;p;}' ;;
-			numerologia) sed -n '/Significado - Numerologia - Expressão/,/ - Arcanos do Tarot/{/Arcanos do Tarot/d;p;}' ;;
-			tarot      ) sed -n '/ - Arcanos do Tarot/,/VEJA TAMBÉM/{/VEJA TAMBÉM/d;p;}' ;;
-			tudo       ) sed -n '/Qual a origem do nome /,/VEJA TAMBÉM/{/VEJA TAMBÉM/d;p;}' ;;
-			*          ) sed -n '/Qual \(a origem\|o significado\) do nome /p' ;;
+			letra      ) sed -n '/ - Analise da Primeira Letra do Nome:/,/<h3>/{/<h3/d;p;}' ;;
+			marca      )
+				sed -n '
+					/ - Analise da Primeira Letra do Nome:/,/Significado - Numerologia - Expressão / {
+						/<h3>/,$!d
+						/<h3>/ i Sua Marca no Mundo
```

Comment:
Usar o `i` sem o escape da quebra de linha, infelizmente não é portável

```suggestion
						/<h3>/ i \
Sua Marca no Mundo
```

Só que daí estraga toda a identação do código :(